test/deep_stacks.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+CONFIG_DEEP_STACK="../test/configs/deep_stacks.toml"
+
+./ddprof --config ${CONFIG_DEEP_STACK} ./test/deep_stacks | tee ./log_ddprof.txt
+
+# Check for truncated stack traces
+truncated_input=$(awk -F': ' '/datadog.profiling.native.unwind.stack.truncated_input/ { print $NF }' ./log_ddprof.txt)
```

Comment:
Why not test that the stacks are correct ?

Reply:
I would need to open source prof-correctness to avoid duplicating some of the logics for this.

Reply:
Yes, but in its current state this test does not test much (and is quite complicated).
In release mode, everything is probably inlined in `main` function.

Reply:
Trying out release mode, everything looks OK
![image](https://github.com/DataDog/ddprof/assets/74836499/92492727-179f-40df-8ede-dcff60289339)

Reply:
I did add a "no inline" attribute to make sure. Thanks for mentioning this.